locksmith/src/utils/getEventOrganizers.ts

```diff
@@ -0,0 +1,38 @@
+import { PaywallLocksConfigType } from '@unlock-protocol/core'
+import { getCheckoutConfigById } from '../operations/checkoutConfigOperations'
+import { getEventBySlug } from '../operations/eventOperations'
+import { isEmpty } from 'lodash'
+import networks from '@unlock-protocol/networks'
+import { Web3Service } from '@unlock-protocol/unlock-js'
+
+export const isEventOrganizer = async (address: string, slug: string) => {
```

Comment:
Actually now I wonder if we have that code somewhere else already? Can you check? If so, it's ok to keep it here, or use the function but please make sure we do not have duplicated code.